mne/datasets/utils.py

```diff
@@ -325,6 +328,7 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
         testing='1ef691944239411b869b3ed2f40a69fe',
         multimodal='26ec847ae9ab80f58f204d09e2c08367',
         fnirs_motor='c4935d19ddab35422a69f3326a01fef8',
+        boxy_example='b3793334548b7ba04c1b767c66117414',
```

Comment:
@rob-luke - I am stuck here in loading a dataset, I uploaded to https://osf.io/hksme/?show=revision, and copied the hash from there, but each time I run plot_80_boxy_processing.py it finds a new hash from the file not this one it is looking for, bring back any memories?

Reply:
I just tried to manually download the OSF file and it said I don't have access. Is the data made public?

Reply:
that is probably the issue, thanks,

Reply:
hey I think that worked, thanks!